src/javascript/binary/base/header.js

```diff
@@ -196,6 +196,7 @@ const Header = (() => {
 
             const messages = {
                 authenticate   : () => buildMessage('[_1]Authenticate your account[_2] now to take full advantage of all payment methods available.',     'user/authenticate'),
+                under_review   : () => buildMessage('We are reviewing your account. For more details [_1]contact us[_2].',                                'contact'),
```

Comment:
Please add the new string to `scripts/bin/GenerateStaticData.pm`.

Reply:
:+1: